test/hotspot/gtest/nmt/test_vmatree.cpp

```diff
@@ -724,3 +759,103 @@ TEST_VM_F(NMTVMATreeTest, TestConsistencyWithSimpleTracker) {
     }
   }
 }
+
+TEST_VM_F(NMTVMATreeTest, SetFlag) {
+  // The gc/cds case with only reserved data
+  {
+    VMATree::SummaryDiff diff;
+    Tree::RegionData rd(NCS::StackIndex(), mtNone);
+    VMATree tree;
+    diff = diff.apply(tree.reserve_mapping(0, 500, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = diff.apply(tree.reserve_mapping(500, 100, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = diff.apply(tree.set_tag(0, 500, mtGC));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = diff.apply(tree.set_tag(500, 100, mtClassShared));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+  }
+
+  // Now let's add in some committed data
+  {
+    VMATree::SummaryDiff diff;
+    Tree::RegionData rd(NCS::StackIndex(), mtNone);
+    VMATree tree;
+    diff = diff.apply(tree.reserve_mapping(0, 500, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = diff.apply(tree.reserve_mapping(500, 100, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    // The committed areas
+
+    diff = diff.apply(tree.commit_mapping(100, 125, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+    diff = diff.apply(tree.commit_mapping(550, 10, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = diff.apply(tree.commit_mapping(565, 10, rd));
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = tree.set_tag(500, 100, mtClassShared);
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+
+    diff = tree.set_tag(0, 500, mtGC);
+    diff.print_self();
+    tree.print_self();
+    tty->cr();
+  }
+}
```

Comment:
This looks like a test which was added when you merged in an earlier version of https://github.com/openjdk/jdk/pull/20994

This should be removed.

Reply:
Done.